openvpn/common/hexstr.hpp

```diff
@@ -180,7 +180,7 @@ namespace openvpn {
 	dest.push_back((high<<4) + low);
       }
     if (i != len)
-      throw parse_hex_error(); // straggler char      
+      throw parse_hex_error(); // straggler char
```

Comment:
remove that change or just create another commit with this change only. If at some point we have to revert a change due to a bug, we don't want to revert other unrelated changes.